src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -44,7 +97,7 @@ public class OtlpExporterOptions
         public int TimeoutMilliseconds { get; set; } = 10000;
 
         /// <summary>
-        /// Gets or sets the export processor type to be used with the OpenTelemetry Protocol Exporter. The default value is <see cref="ExportProcessorType.Batch"/>.
+        /// Gets or sets the export processor type to be used with the OpenTelemetry Protocol Exporter. The default value is <see cref="Batch{T}"/>.
```

Comment:
Why make this change? I think the old version is correct.

Reply:
Seems like I got it done by some automatic refactoring. Thanks for catching that